src/defold/support/ScriptMacro.hx

```diff
@@ -393,7 +396,9 @@ private class Glue {
             case [PVector4, [{expr: EConst(CFloat(x) | CInt(x))}, {expr: EConst(CFloat(y) | CInt(y))}, {expr: EConst(CFloat(z) | CInt(z))}, {expr: EConst(CFloat(w) | CInt(w))}]]:
                 'vmath.vector4($x, $y, $z, $w)';
             case [PQuaternion, [{expr: EConst(CFloat(x) | CInt(x))}, {expr: EConst(CFloat(y) | CInt(y))}, {expr: EConst(CFloat(z) | CInt(z))}, {expr: EConst(CFloat(w) | CInt(w))}]]:
-                'vmath.quat($x, $y, $z, $w)';
+				'vmath.quat($x, $y, $z, $w)';
+			case [PBufferResourceReference, [{expr: EConst(CString(s))}]]:
+				'resource.buffer(${haxe.Json.stringify(s)})';
```

Comment:
Indentation

Reply:
Also move the `PBufferResourceReference` to be the last to be consistent with the declaration.